src/optimizer/query_graph.h

```diff
@@ -571,6 +571,7 @@ typedef enum
 #define QO_IS_PATH_TERM(t)	(QO_TERM_CLASS(t) & 0x20)
 #define QO_IS_EDGE_TERM(t)	(QO_TERM_CLASS(t) & 0x10)
 #define QO_IS_FAKE_TERM(t)	(QO_TERM_CLASS(t) & 0x08)
+#define QO_IS_DEP_TERM(t)	(QO_TERM_CLASS(t) == QO_TC_DEP_LINK || QO_TERM_CLASS(t) == QO_TC_DEP_JOIN)
```

Comment:
QO_IS_FAKE_TERM is used in several optimizing routines.
needed to check whether applying to the routines is correct.

Reply:
I have already checked and no changes are required.
The function that invoke 'QO_IS_FAKE_TERM' is as follows.
```
make_pred_from_bitset() : Condition for not creating predicate when creating xasl. No dummy join term should be generated.
gen_outer  : Condition to delete subquery. dummy join term has no subquery.
get_term_rank : Condition not to create rank. dummy join term does not require rank. (rank is used for the same selectivity)
```